modyn/supervisor/internal/pipeline_executor/pipeline_executor.py

```diff
@@ -651,11 +646,19 @@ def _train(self, s: ExecutionState, log: StageLog, trigger_id: int) -> int:
         s.current_training_id = self.grpc.start_training(
             s.pipeline_id,
             trigger_id,
-            s.pipeline_config.model_dump(by_alias=True),
+            s.pipeline_config.training,
+            s.pipeline_config.data,
             s.previous_model_id,
             num_samples_to_pass,
         )
-        trainer_log = self.grpc.wait_for_training_completion(s.current_training_id, s.pipeline_id, trigger_id)
+
+        total_samples = self.grpc.get_number_of_samples(s.pipeline_id, trigger_id)
+        status_bar_scale = self.grpc.get_status_bar_scale(s.pipeline_id)
+        training_reporter = TrainingStatusReporter(
+            self.state.training_status_queue, trigger_id, s.current_training_id, total_samples, status_bar_scale
+        )
```

Comment:
Why did you move the training status reporter initialization out of `wait_for_training_completion`? If I see correctly, it used to be in there (including status bar scale etc)